tests/func/plots/test_show.py

```diff
@@ -827,3 +827,13 @@ def test_show_dir_plots(tmp_dir, dvc, run_copy_metrics):
 
     result = dvc.plots.show(targets=[p1])
     assert set(result.keys()) == {p1}
+
+
+def test_ignore_binary_file(tmp_dir, dvc, run_copy_metrics):
+    with open("file", "wb") as fobj:
+        fobj.write(os.urandom(124))
```

Comment:
There might be a chance that this will generate something that can be decoded as text. Better to just hardcode a particular blob here. It doesn't even need to be long.